app/views/about/_small_and_fast.html.erb

```diff
@@ -89,22 +89,22 @@
-        <tr><td nowrap="">Update</td><td class="desc">Pull of Commit A scenario (113 files changed, 2164+, 2259-)</td><td class="number"> 0.90</td><td class="number"> 2.82</td><td class="number">3x</td></tr>
-        <tr><td nowrap="">Blame</td><td class="desc">Line annotation of a single file (array.c)</td><td class="number"> 1.91</td><td class="number"> 3.04</td><td class="number">1x</td></tr>
+        <tr><td nowrap="">Commit Files (A)</td><td class="desc">Add, commit and push 113 modified files (2164+, 2259-)</td><td class="number">0.64</td><td class="number">2.60</td><td class="number">4x</td></tr>
+        <tr><td nowrap="">Commit Images (B)</td><td class="desc">Add, commit and push 1000 1&nbsp;kB images</td><td class="number">1.53</td><td class="number">24.70</td><td class="number">16x</td></tr>
+        <tr><td nowrap="">Diff Current</td><td class="desc">Diff 187 changed files (1664+, 4859-) against last commit</td><td class="number">0.25</td><td class="number">1.09</td><td class="number">4x</td></tr>
+        <tr><td nowrap="">Diff Recent</td><td class="desc">Diff against 4 commits back (269 changed/3609+,6898-)</td><td class="number">0.25</td><td class="number">3.99</td><td class="number">16x</td></tr>
+        <tr><td nowrap="">Diff Tags</td><td class="desc">Diff two tags against each other (v1.9.1.0/v1.9.3.0)</td><td class="number">1.17</td><td class="number">83.57</td><td class="number">71x</td></tr>
+        <tr><td nowrap="">Log (50)</td><td class="desc">Log of the last 50 commits (19&nbsp;kB of output)</td><td class="number">0.01</td><td class="number">0.38</td><td class="number">31x</td></tr>
+        <tr><td nowrap="">Log (All)</td><td class="desc">Log of all commits (26,056 commits &ndash; 9.4&nbsp;MB of output)</td><td class="number">0.52</td><td class="number">169.20</td><td class="number">325x</td></tr>
+        <tr><td nowrap="">Log (File)</td><td class="desc">Log of the history of a single file (array.c &ndash; 483 revs)</td><td class="number">0.60</td><td class="number">82.84</td><td class="number">138x</td></tr>
+        <tr><td nowrap="">Update</td><td class="desc">Pull of Commit A scenario (113 files changed, 2164+, 2259-)</td><td class="number">0.90</td><td class="number">2.82</td><td class="number">3x</td></tr>
+        <tr><td nowrap="">Blame</td><td class="desc">Line annotation of a single file (array.c)</td><td class="number">1.91</td><td class="number">3.04</td><td class="number">1x</td></tr>
       </tbody>
     </table>
-    
+
     <p>
-      Note that this is the best case scenario for SVN - a server with no load with an
-      80MB/s bandwidth connection to the client machine.  Nearly all of these
+      Note that this is the best case scenario for SVN&mdash;a server with no load with an
+      640&nbsp;Mbit/s bandwidth connection to the client machine. Nearly all of these
```

Comment:
I'm assuming this was meant to be 80 megabytes per second before, and not 80 megabits per second.

Reply:
Yes, probably. The commit that introduced it (98c7d259136268076f548bd02c8fce5229f314b2) is light on details, but I'd think that a gigabit connection would have been "best case" in 2012 (and with some overhead, would probably allow 80 megabytes or better). Maybe it would make sense to just say "...with a gigabit connection", since the exact bandwidth number isn't really relevant?

Reply:
Well given that the exact times are quoted in the table, it would make sense to be exact about the bandwidth as well.

Reply:
But most of those are not bandwidth-limited in the first place (and I'm not sure we actually know the exact bandwidth anyway).

Reply:
Interesting. Sounds good, I'll change it then.